content/_data/events/2018-02-15-austin.adoc

```diff
@@ -0,0 +1,7 @@
+---
+name: "Austin JAM"
```

Comment:
The reason that the Jenkins run is failing is because this should be `title` not `name` FYI